avocado/utils/disk.py

```diff
@@ -170,3 +171,72 @@ def is_root_device(device):
         if item['mountpoint'] == "/" and device == str(item['pkname']):
             return True
     return False
+
+
+def is_disk_mounted(device):
+    """
+    check if given disk is mounted or not
+    param: disk/device name
+    Returns True if the device/disk is mounted else False
+    """
+    with open('/proc/mounts') as mounts:
+        for mount_line in mounts.readlines():
+            dev, fs_dir, fs_vfstype, mount_op, _, _ = mount_line.split()
+            if dev == device:
+                return True
+        return False
+
+
+def is_dir_mounted(dir_path):
+    """
+    check if given directory is mounted or not
+    param: directory path
+    return: True if the given director is mounted else False
+    """
+    with open('/proc/mounts') as mounts:
+        for mount_line in mounts.readlines():
+            dev, fs_dir, fs_vfstype, mount_op, _, _ = mount_line.split()
+            if fs_dir == dir_path:
+                return True
+        return False
+
+
+def is_fs_exists(device):
```

Comment:
`is_fs_exists`? or `is_fs_existent`? or just `fs_exists`?

Reply:
I feel `is_fs_exists` is OK.
but people can suggest the best if feel so.

Reply:
Not a native speaker, but to the best of knowledge, `is_fs_exists` is a wrong grammatical construct.  I'd suggest `fs_exists` or even `is_fs` (if something *is*, it *exists*).

Reply:
@beraldoleal  `fs_exists`  looks good to me. will change this method name.
only this method name you want me to change or even other methods in this PR?

Reply:
@clebergnu @beraldoleal any info here?
most of our code testing is waiting for these changes. 
early help is really appreciated :-)